ui/src/app/assets/components/asset-overview/asset-overview.component.ts

```diff
@@ -147,4 +150,26 @@ export class SpAssetOverviewComponent implements OnInit {
                 this.loadAssets();
             });
     }
+
+    downloadAsset(asset: SpAssetModel) {
+        this.dataExportService
+            .getExportPreview([asset._id])
+            .pipe(
+                mergeMap(preview =>
+                    this.dataExportService.triggerExport(preview),
+                ),
+            )
+            .subscribe((data: Blob) => {
+                const blob = new Blob([data], { type: 'application/zip' });
```

Comment:
Hi @IsaakKrut,
Is it also possible to use the file-saver API here? If not, the existing approach is ok for me.

Reply:
Updated to use file-saver